metadata.json

```diff
@@ -11,30 +11,26 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "6",
```

Comment:
I'm fine with this as our tests can't pass otherwise.  Our general policy might also be to remove EL6 related code from modules, but since this is the EPEL module, I'm for leaving such code in if we can.

Reply:
@alexjfisher I was originally going to do that but then figured there could be people still using with EL6 so didn't want to break that for this change just wanted to fix the broken CI. If going to remove EL6 support completely it probably would be a bigger update and not sure how you'd track something like that as breaking vs this that's minor feature/patch release?